### PR TITLE
restic restore helper image can be configured with cli flag

### PR DESCRIPTION
**Pull request by ryane**
This is an attempt to address #638. I [also](https://github.com/heptio/velero/issues/638#issuecomment-437639874) did not find it straightforward to pass the image from a velero server argument to the plugin command. So, I *tried* to keep changes to a minimum and to just get something working but I'm sure this could be improved. I'd be happy to continue to refine this PR but would love to get some feedback before spending too much time on it. I'd really just like to help get a fix for this so that we can use an official velero build in our on-prem, air gapped environment.

### CODE REVIEW

File: pkg/restore/restic_restore_action.go [hunk 1]
```diff
@@ -36,10 +36,14 @@ type ResticRestoreAction struct {
 	initContainerImage string
 }
 
-func NewResticRestoreAction(logger logrus.FieldLogger) *ResticRestoreAction {
+func NewResticRestoreAction(logger logrus.FieldLogger, initContainerImageName string) *ResticRestoreAction {
+	if initContainerImageName == "" {
```
Comment (nrb) [inline pkg/restore/restic_restore_action.go]: I'll have to think a bit more on how to accomplish this, but it'd be awesome if the `initContainerImage` function could just return the image from the args.

**Comment by skriss** [comment]
Thanks for this @ryane. We should consider the approach here as part of #584 - do we want to use ConfigMaps instead of CLI flags for configuring plugins? We shouldn't have to change code every time a new flag/config option is added. cc @carlisia

**Comment by carlisia** [comment]
This looks good @ryane. If it could be made to use ConfigMap for this plugin config it would be better.

**Comment by skriss** [comment]
@ryane we decided to go with a ConfigMap-based approach since it will be more extensible for generic configuration of all plugins. See #1311 which recently merged. Appreciate the PR regardless!

**Comment by ryane** [comment]
excellent, thx!

**Pull request closed without merging**